src/dataCorrectionsTool/surveyResetTool.js

```diff
@@ -122,48 +124,61 @@ const handleSurveyTypeChange = (participant) => {
     if (!surveyDropdown || !dropdownSurveyOptions || !selectButton) return;
 
     const participantConnectId = participant['Connect_ID'];
+    const { ssnStatusFlag, notStarted } = fieldMapping;
     let query;
 
     for (let option of dropdownSurveyOptions) {
         option.addEventListener('click', async (e) => {
-            selectedSurvey = e.target.dataset.survey;
-            if (selectedSurvey === 'ssn') {
+            selectedSurvey = Number(e.target.dataset.survey);
```

Comment:
Maybe just for the future: it would be good to guard against the potential of duplicate event listeners being created if the user hits this dropdown multiple times. I think that could be a source of some of our unexpected double-click issues (but I'm not certain).

Reply:
Thanks for the suggestion. I'm going to add a todo comment